docs/Installing.asciidoc

```diff
@@ -748,6 +748,35 @@ keys_to_render_as_links=FOO,AUTOYAST
 The files referenced by the configured keys should be located either under the root
 of `CASEDIR` or the data folder within `CASEDIR`.
 
+=== Enable custom hook scripts on "job done" based on result
+
+If a job is done, especially if no label could be found for carry-over, often
+more steps are needed for the review of the test result or providing the
+information to either external systems or users. As there can be very custom
+requirements openQA offers a point for optional configuration to let the
+instance administrators define specific actions.
+
+For example there is already an approach called "auto-review"
+https://github.com/os-autoinst/scripts/#auto-review---automatically-detect-known-issues-in-openqa-jobs-label-openqa-jobs-with-ticket-references-and-optionally-retrigger
+which offers helpful, external scripts. By setting custom hook scripts it is
+possible to call external scripts if defined and based on the selected either
+environment variables or config settings. For example config settings for
+openqa.opensuse.org enabling the auto-review scripts could look like:
+
+```
+[global]
+JOB_DONE_HOOK_INCOMPLETE = /opt/openqa-scripts/openqa-label-known-issues-hook
+JOB_DONE_HOOK_FAILED = /opt/openqa-scripts/openqa-label-known-issues-hook
+```
+
+or for a host openqa.example.com:
+
+```
+[global]
```

Comment:
It would be nicer if these settings had their own section.

Reply:
ok, I selected `[hooks]`